articles/virtual-machines/error-codes-spot.md

```diff
@@ -20,6 +20,7 @@ Here are some possible error codes you could receive when using Azure Spot Virtu
 | Key | Message | Description |
 |-----|---------|-------------|
 | SkuNotAvailable | The requested tier for resource '\<resource\>' is currently not available in location '\<location\>' for subscription '\<subscriptionID\>'. Try another tier or deploy to a different location. | There is not enough Azure Spot Virtual Machine capacity in this location to create your VM or scale set instance. |
+| Standard_XX-XX_XX. This size does not support Azure Spot.|  There is insufficient Azure Spot Virtual Machine capacity in this location to create your VM or scale set instance.|  For Spot instances, if there is no available capacity, the portal page will fail immediately. This fail-fast feature ensures that users are promptly informed of the allocation failure, preventing them from waiting for an error after attempting deployment. 
```

Comment:
```suggestion
| Standard_XX-XX_XX. This size does not support Azure Spot.|  There is insufficient Azure Spot Virtual Machine capacity in this location to create your VM or scale set instance.|  For Spot instances, if there is no available capacity, the portal page will fail immediately. This fail-fast feature ensures that users are promptly informed of the allocation failure, preventing them from waiting for an error after attempting deployment. |
```

Reply:
@mzakirs Can you review this edit and commit it if you agree? Let us know if you need any assistance.